.github/workflows/windows.yml

```diff
@@ -0,0 +1,52 @@
+name: Build and Package
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  build:
+    runs-on: windows-latest
+
+    strategy:
+      matrix:
+        vs-version:
+          - '2013'
+          - '2015'
+          - '2017'
+          - '2019'
+          - '2022'
+        configuration:
+          - 'Debug'
+          - 'Release'
+        architecture:
+          - 'x86'
+          - 'x64'
```

Comment:
I believe Github only allows 15 concurent jobs defined by `matrix`
here we have 20
that is either has to be split into different jobs or drop some of the configuration combinations  (e.g. run Debug only for x64)

Reply:
> I believe Github only allows 15 concurent jobs defined by `matrix`

Huh, TIL, thanks.

> drop some of the configuration combinations (e.g. run Debug only for x64)

Yeah to me it seems sensible to drop Release for most modes and add only one we want to package, but for now I was trying to migrate what Appveyor did 1:1 (minus the Appveyor part).

Reply:
unless libusb Org has different limitations...
all 20 seem to spawn just fine, resolving for now